python/tvm/relay/frontend/pytorch.py

```diff
@@ -183,9 +183,27 @@ def _impl(inputs, input_types):
         data = inputs[0]
         output_size = _infer_shape(inputs[1])
 
+        # returns dummy indices too
         return _op.nn.adaptive_max_pool2d(
             data,
-            output_size=output_size)
+            output_size=output_size), None
+    return _impl
+
+def _adaptive_max_3d():
+    def _impl(inputs, input_types):
+        data = inputs[0]
+        output_size = _infer_shape(inputs[1])
+        # returns dummy indices too
+        return _op.nn.adaptive_max_pool3d(data, output_size=output_size), None
```

Comment:
Does adaptive pooling in pytorch always return two values? The documentation seems to imply that it only returns indices if the `return_indices` parameters is `True`.

Reply:
That is for python layer, after jitting torchscript `aten::adaptive_max_pool3d` (and 2d) op always returns 2 results. If `return_indices` is False, I think the second tensor is empty.

See 
https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/AdaptiveMaxPooling3d.cpp#L421